.travis.yml

```diff
@@ -21,5 +22,6 @@ cache:
   yarn: true
 
 script:
+  - yarn run build
```

Comment:
Avoid building everything, simply change the script `prepublishOnly` to `prepare` for packages needed for the tests.